src/tests/blueprint/t_blueprint_o2mrelation_examples.cpp

```diff
@@ -187,7 +190,7 @@ TEST(conduit_blueprint_o2mrelation_examples, o2mrelation_generate_offsets)
     n.remove("offsets");
     EXPECT_TRUE(blueprint::o2mrelation::generate_offsets(n, info));
     EXPECT_TRUE(blueprint::o2mrelation::verify(n,info));
-    EXPECT_FALSE(ref.diff(n, info));
+    EXPECT_FALSE(ref.diff(n, info, epsilon, relaxint));
```

Comment:
This was returning true because `n` had sizes of type uint32, not index_t == int64.